restlicsharpclient/restliclient/util/RequestUrlBuilder.cs

```diff
@@ -19,6 +19,7 @@
 
 using restlicsharpclient.restliclient.request;
 using restlicsharpclient.restliclient.response;
+using System.Linq;
 
 namespace restlicsharpclient.restliclient.util
```

Comment:
Maybe we should put these url related class into a separate namespace, for example "url" instead of existing "util".